elasticsearch/README.md

```diff
@@ -0,0 +1,17 @@
+## 0x00
+`sudo -H pip3 install elasticsearch`.
+
+## 0x01
+Download Elasticsearch and Kibana (works with 6.0.0, going to try with newest) from [here](https://www.elastic.co/downloads/past-releases)
```

Comment:
"going to try with newest" is not something I would like to see in a proper documentation file like this.